core/src/test/resources/sql/unnest.iq

```diff
@@ -105,6 +105,29 @@ from UNNEST(array ['apple', 'banana']) as "fruit";
 
 !ok
 
+select fruit
+from UNNEST(array(select 'banana')) as fruit;
++--------+
+| FRUIT  |
++--------+
+| banana |
++--------+
+(1 row)
+
+!ok
+
+#TODO: Uncomment when CALCITE-5127 will be fixed
```

Comment:
@dssysolyatin Please remove the comments and use `!if (false) {` instead. (Results in fewer diffs.) Also change 'CALCITE-5127' to '[CALCITE-5127]' for easier search.